readthedocs/doc_builder/python_environments.py

```diff
@@ -245,7 +245,7 @@ def install_core_requirements(self):
                     positive='sphinx<2',
                     negative='sphinx==1.7.4',
                 ),
-                'sphinx-rtd-theme<0.3',
+                'sphinx-rtd-theme==0.3.1',
```

Comment:
This should not be pinned to a specific version, so we can release & ship new versions. It should likely bump to `<0.4`.